pageserver/src/metrics.rs

```diff
@@ -505,7 +502,8 @@ const STORAGE_IO_TIME_OPERATIONS: &[&str] = &[
 
 const STORAGE_IO_SIZE_OPERATIONS: &[&str] = &["read", "write"];
 
-pub static STORAGE_IO_TIME: Lazy<HistogramVec> = Lazy::new(|| {
+// Needed for https://neonprod.grafana.net/d/8G011dlnk/timeline-inspector?orgId=1
```

Comment:
Missed that it's actually fine to make this global.

Reply:
Found the missing `drop` with this :)

Reply:
[a053b96](https://github.com/neondatabase/neon/pull/4769/commits/a053b96640b17534925e63d0337d89d3697a6784)